src/components/elements/Video.js

```diff
@@ -6,16 +6,16 @@ import withLazyLoad from './withLazyLoad'
 const IframeWrapper = styled.div`
   > iframe {
     border: 0;
-    height: 390px;
+    ${props => `height: ${props.height}`}px;
   }
   > div {
     margin-top: -15px;
     margin-bottom: 20px;
   }
 `
 
-const Video = ({ youtubeId, time, description }) => (
-  <IframeWrapper>
+const Video = ({ youtubeId, time, description, height = '390' }) => (
+  <IframeWrapper height={height}>
```

Comment:
This is something we'll need for the next PR as all pics are being replaced with Pau's new videos and we need to be able to adjust the height in some cases